### PR TITLE
[READY]Plague rat rework and expansion

### PR DESCRIPTION
**Pull request by ghost**
# Document the changes in your pull request

At first, xenos now can have plague, but no more are affected by it.

At second, plague rats now have 3% chance to spawn when mice migration because 5 is too much.

At third, no more plague rats, they are now disease rats. It means that they get a random disease from a list, and need to spread it.

[The fourth part should be making them have an antag datum, but cry about it.]

At fifth - new reagent. Honkpax. It makes the guy who has it in blood system clumsy(temporary).

At sixth - new disease - Brain honk - produces honkpax into its victim, makes them honk randomly, get minor brain damage and randomly slip on imaginable bananas. Also if caried by a plague rat makes it honk when attacking, and makes it slippery.

Also they will be some random balance changes.

# Wiki Documentation

Document some of this shit

# Changelog

Edit the changelog below to reflect the changes made by this PR, even if the changes are minor - required for every PR.
If you add a name after the ':cl', that name will be used in the changelog. Leave it empty to use your GitHub name.

:cl:  
rscadd: Reworks plague rats into disease rats
/:cl:

### CODE REVIEW

File: code/datums/diseases/plague.dm [hunk 1]
```diff
@@ -15,6 +15,8 @@
 
 /datum/disease/plague/stage_act()
 	..()
+	if(istype(affected_mob, /mob/living/carbon/alien))
```
Comment (tattax) [inline code/datums/diseases/plague.dm]:
```suggestion
	if(isalien(affected_mob))
```

File: code/modules/shuttle/computer.dm [hunk 2]
```diff
@@ -137,3 +137,7 @@
 /obj/machinery/computer/shuttle/connect_to_shuttle(obj/docking_port/mobile/port, obj/docking_port/stationary/dock, idnum, override=FALSE)
 	if(port && (shuttleId == initial(shuttleId) || override))
 		shuttleId = port.id
+
+/obj/machinery/computer/shuttle/AltClick(user)
+	if(istype(user, /mob/living/carbon) || istype(user, /mob/living/simple_animal/hostile/asteroid/elite))
+		ui_interact(user, null)
```
Comment (tattax) [inline code/modules/shuttle/computer.dm]:
```suggestion
```

**Review by tattax — CHANGES_REQUESTED** [review]

**Comment by ghost** [comment]
Okay i suddenly don't want to work more on this shitty PR, it is finnished

File: code/modules/mob/living/simple_animal/hostile/rat.dm [hunk 3]
```diff
@@ -167,9 +183,11 @@
...
 				var/obj/item/I = C.get_item_by_slot(ITEM_SLOT_OCLOTHING)
 				if(!istype(I, /obj/item/clothing/suit/space/hardsuit) && !istype(I, /obj/item/clothing/suit/armor) && !istype(I, /obj/item/clothing/suit/bio_suit/plaguedoctorsuit))
-					var/datum/reagent/plaguebacteria/plague = new /datum/reagent/plaguebacteria
-					C.reagents.add_reagent(/datum/reagent/plaguebacteria, 3) //gets caught up in blood transfers
-					plague.reaction_mob(C, INJECT) //making sure we stay winning
+					var/datum/reagent/diseaseshit = new reagent
+					C.reagents.add_reagent(reagent, 3) //gets caught up in blood transfers
+					diseaseshit.reaction_mob(C, INJECT) //making sure we stay winning
+				if(disease == "Brain Honk")
+					playsound(get_turf(affected_mob), 'sound/items/bikehorn.ogg', 50, TRUE)
```
Comment (tattax) [inline code/modules/mob/living/simple_animal/hostile/rat.dm]:
```suggestion
					playsound(get_turf(C), 'sound/items/bikehorn.ogg', 50, TRUE)
```

**Comment by ToasterBiome** [comment]
fix

**Comment by TymurShatillo** [comment]
Ratio

**Pull request closed without merging**